base/workflow/R/run.write.configs.R

```diff
@@ -211,13 +228,26 @@ run.write.configs <- function(settings, ensemble.size, input_design, write = TRU
   PEcAn.logger::logger.info("###### Finished writing model run config files #####")
   PEcAn.logger::logger.info("config files samples in ", file.path(settings$outdir, "run"))
 
-  ### Save output from SA/Ensemble runs
-  # A lot of this is duplicate with the ensemble/sa specific output above, but kept for backwards compatibility.
-  save(ensemble.samples, trait.samples, sa.samples, runs.samples, pft.names, trait.names,
-    file = file.path(settings$outdir, "samples.Rdata")
-  )
-  PEcAn.logger::logger.info("parameter values for runs in ", file.path(settings$outdir, "samples.RData"))
+  # write runs manifest
+
+  manifest.file <- file.path(settings$outdir, "runs_manifest.csv")
+
+  if (nrow(run_manifest_df) > 0) {
+    if (overwrite && file.exists(manifest.file)) {
+      unlink(manifest.file)
+    }
+
+    utils::write.table(run_manifest_df,
+                       file = manifest.file,
+                       sep = ",",
+                       row.names = FALSE,
+                       col.names = !file.exists(manifest.file),
+                       append = file.exists(manifest.file))
+
+    PEcAn.logger::logger.info("Run manifest written to ", manifest.file)
+  }
+
   options(scipen = scipen)
   invisible(settings)
   return(settings)
```

Comment:
Not your fault but fixing this while I see it -- `invisible()` was doing nothing here and usually only makes sense as a return value
```suggestion
  return(invisible(settings))
```

Reply:
I should know this, but what's actually been changed in the returned settings object. I'm wondering whether the settings is still the right thing to return at this point (e.g., are we moving to a place where the manifest might be the thing a user actually wants back?)

Reply:
Changed: at least ensemble IDs and PFT outdirs (if they weren't there in the input settings).

Return value: My gut is settings is still the right thing to return. I'm saying this without checking, but I think at the moment every `runModule.*()` takes a MultiSettings and returns a possibly modified MultiSettings, so changing that would be an even bigger overhaul. But maybe this is an argument for putting (a copy of) the manifest _into_ the returned settings?

Reply:
> But maybe this is an argument for putting (a copy of) the manifest into the returned settings?

If you are saying (settings$runs_manifest <- run_manifest_df), then manifest written to pecan.CONFIGS.xml via listToXml(), which would serialize every row as nested XML elements. But for for large SDA runs would increase the disk size (may be in giga byte). Slow down read.settings() / write.settings()